src/app/technical-record/body/edit/body-edit.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
```

Comment:
replace to this `  <div class="govuk-grid-column-three-quarters">`

Reply:
done